.github/workflows/optimism-testing.yml

```
@@ -0,0 +1,39 @@
+name: Optimism testing
+
+on:
+  push:
+    branches:
+      - changeset-release/*
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    strategy:
        matrix:
            OPTIMISM_GIT_HEAD: ["master", "b0a54509a6f99194221f5ade230ceae7892b3393"]
```

(test against a known version that's working right now, and a current `master`